src/java/voldemort/common/service/SchedulerService.java

```diff
@@ -229,18 +229,22 @@ public SchedulerThreadPool(int numThreads, final String schedulerName) {
                 private AtomicInteger threadCount = new AtomicInteger(0);
 
                 /**
-                 * This function is overridden in order to activate the daemon mode as well as
-                 * to give a human readable name to threads used by the {@link SchedulerService}.
-                 * 
-                 * Previously, this function would set the thread's name to the value of the passed-in
-                 * {@link Runnable}'s class name, but this is useless since it always ends up being a
```

Comment:
Nope my eclipse is messing it up. Though I have only format changed line settings.